NEWS.md

```diff
@@ -91,6 +91,7 @@
         replacement of `pl$reset_options()`).
 -   New function `polars_envvars()` to print the list of environment variables
     related to polars (#735).
+-   New method `$write_parquet()` for DataFrame (#745).
```

Comment:
Use the PR number instead of the issue number. You can also add your github tag after the PR number if you want